.drone.yml

```diff
@@ -215,6 +256,27 @@ steps:
     event:
     - tag
 
+- name: e2e-tests
+  image: plugins/manifest:1.0.2
+  settings:
+    username:
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+    platforms:
+      - linux/amd64
+      - linux/arm64
```

Comment:
Is there a reason arm is not included here?

Reply:
there is currently no sonobuoy client built for arm